test/library/oc.go

```
@@ -0,0 +1,103 @@
+	out, err := cmd.run()
+	return string(out), string(out), err
+}
+
+func (cmd *ocCommand) run() ([]byte, error) {
+	args := append([]string{}, cmd.args...)
+	if !cmd.oc.noNS && cmd.oc.namespace != "" {
+		args = append(args, "-n", cmd.oc.namespace)
+	}
+	full := append([]string{"oc"}, args...)
+	fmt.Fprintf(g.GinkgoWriter, "running: %s\n", strings.Join(full, " "))
+	c := exec.Command("oc", args...)
```

golangci-lint (2.12.2)

[error] os/exec.Command must not be called. use os/exec.CommandContext

(noctx)